Formula/t.rb

```diff
@@ -0,0 +1,18 @@
+class T < Formula
+  desc "Command-line todo list manager for finishing tasks, not organizing them"
+  homepage "https://stevelosh.com/projects/t/"
+  url "https://github.com/sjl/t/archive/v1.2.0.tar.gz"
```

Comment:
The github is a mirror, not the official source.

Reply:
The upstream is a Mercurial repository and does not, as far as I can tell, provide source tarballs for tagged releases. I used the Github mirror in order to avoid an `hg` dependency as well as to be able to use the tarball.